codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyFrameCodec.java

```diff
@@ -407,4 +417,20 @@ public void readHeaderBlockEnd() {
     public void readFrameError(String message) {
         ctx.fireExceptionCaught(INVALID_FRAME);
     }
+
+    protected void setRead(final boolean read) {
```

Comment:
can you show me a snippet how you would use this one ?

Reply:
```java
    @Override
    public void readUnknownFrame(final int frameType, final byte flags, final ByteBuf payload) {
        setRead(true);
        final byte[] data = ByteBufUtil.getBytes(payload);
        final CustFrame custFrame = newCustomFrame((short) frameType, flags, data);
        getChannelHandlerContext().fireChannelRead(custFrame);
    }
```

Reply:
Couldn't we for example just let it return true if we consider something was read ?

Reply:
But I can only do that when this PR target 4.2, because the `read` is private and I can't add method to the `SpdyFrameDecoderDelegate`

Reply:
yeah that's why I asked before about 4.2. Lets do the PR against it